test/v2/client/test_client.go

```diff
@@ -661,23 +626,21 @@ func (c *TestClient) ReadBlobFromValidatorsInQuorum(
 		defer cancel()
 	}
 
+	start := time.Now()
```

Comment:
this change looks like another merge mistake, presumably `start` should be defined inside the `validateAndDecode` block